pkg/binding/binding.go

```diff
@@ -22,46 +22,36 @@ import (
 
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/klog/v2"
 
 	"github.com/kubestellar/kubestellar/api/control/v1alpha1"
-	"github.com/kubestellar/kubestellar/pkg/util"
 )
 
 // syncBinding syncs a binding object with what is resolved by the bindingpolicy resolver.
-func (c *Controller) syncBinding(ctx context.Context, objIdentifier util.ObjectIdentifier) error {
+func (c *Controller) syncBinding(ctx context.Context, bindingName string) error {
 	logger := klog.FromContext(ctx)
```

Comment:
why the change from objectIdentifier to a bindingName string ? Is this part of some other cleanup effort?

Reply:
I think it is better to make the formal arguments take only the name, not suggest that there is a meaningful namespace input.

I do not think it makes much of a difference whether the conversion from `bindingRef` to string is done inside this method or not. I chose to do it in the logic closer to the workqueue because it seems specific to that.